src/meta/Cargo.toml

```diff
@@ -64,6 +64,7 @@ scopeguard = "1.2.0"
 sea-orm = { workspace = true }
 serde = { version = "1.0.196", features = ["derive"] }
 serde_json = "1.0.113"
+simplestcrypt = "0.1"
```

Comment:
Better to consider a more popular encryption lib

Reply:
https://github.com/TotalKrill/simplestcrypt/blob/master/src/lib.rs
Took a rough look at the crate, the owner just wrapped a basic usage of an audited encryption crate, instead of writing the real "encryption" part. Seems not that vital. What do you think?